src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimeline.cs

```diff
@@ -182,6 +182,8 @@ private async Task RunLoop(CancellationToken cancellationToken)
             _logger.LogTrace("Aggregating results...");
             foreach ((Build build, Task<Timeline> timelineTask) in allNewTimelines)
             {
+                using IDisposable buildScope = _logger.BeginScope(KeyValuePair.Create("buildId", build.Id));
```

Comment:
[not even a nit, no need to change] I really like this new syntax, but this is one case where it looks really weird, declaring an IDisposable variable.

Reply:
Especially in this case where I do not need to access the variable, I just need it to exist and eventually be disposed.